File: AD-HOC/1414/1414.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		while (true) {
			int timesParticipantes = scan.nextInt();
			int qtdPartidas = scan.nextInt();
			int totalPontos = 0;

			if (timesParticipantes == 0 && qtdPartidas == 0)
				break;

			for (int i = 0; i < timesParticipantes; i++) {
				scan.next();
				totalPontos += scan.nextInt();
			}
			
			int empates = qtdPartidas - (totalPontos - (qtdPartidas * 2));

			System.out.println(empates);
		}
		scan.close();
	}
}

File: AD-HOC/1414/1414.cpp
#include <iostream>

using namespace std;

int main()
{
    while(true) {
        string nome;
        int timesParticipantes;
        int qtdPartidas;
        int pontos;
        int totalPontos = 0;
        
        cin >> timesParticipantes;
        cin >> qtdPartidas;
        
        if(timesParticipantes == 0 && qtdPartidas == 0) break;
        
        for(int i=0; i<timesParticipantes; i++) {
            cin >> nome;
            cin >> pontos;
            totalPontos += pontos;
        }
        
        int empates = qtdPartidas - (totalPontos - (qtdPartidas * 2));
        
        cout << empates << "\n";
    }
    return 0;
}